src/components/careers/Main.js

```diff
@@ -13,34 +14,47 @@ const useStyles = createUseStyles((theme) => ({
   },
 }));
 
-const CareersMain = ({ role, requirements, offer, process }) => {
+const CareersMain = ({ showOpenSource, role, requirements, offer, process }) => {
   const classes = useStyles();
 
   return (
     <div className={classes.mainWrapper}>
       <main className={classes.content}>
+        {showOpenSource ? (
+          <LayoutControl maxWidthBreakpoint="lg">
+            <InterstitialTitle text="What's the opportunity? 🤔" />
+            <OpenSourceMission classes={classes} />
+          </LayoutControl>
+        ) : (
+          <LayoutControl maxWidthBreakpoint="lg">
+            <InterstitialTitle text="What is the company's mission? 🤔" />
+            <Mission classes={classes} />
+          </LayoutControl>
+        )}
         <LayoutControl maxWidthBreakpoint="lg">
-          <InterstitialTitle text="Our mission" />
-          <Mission classes={classes} />
-        </LayoutControl>
-
-        <LayoutControl maxWidthBreakpoint="lg">
-          <InterstitialTitle text="The role" />
+          <InterstitialTitle text="What does my day to day look like? 📅" />
           <UnorderedList content={role} />
         </LayoutControl>
 
         <LayoutControl maxWidthBreakpoint="lg">
-          <InterstitialTitle text="The requirements" />
+          <InterstitialTitle text="What skills do I need? ✨" />
```

Comment:
I wonder should we lighten up the language here and make it less of a "need" and more of a "want". It's more inclusive language.

Reply:
Good idea, "What skills would we like to see?"

Reply:
Action: will make it read "What skills would we like to see?"